plugins/cosmos-crypto/.pylintrc

```diff
@@ -0,0 +1,37 @@
+[MASTER]
```

Comment:
This is complete overkill. We have all this duplication now.

Reply:
removed